library/ceph_disks.py

```diff
@@ -1,176 +0,0 @@
-#!/usr/bin/python
```

Comment:
I'd rather not keep the library in this repo if it's not going to be used by code in this repo. That may spur another discussion about shared bits between cloud deploys and cloud operations, but I don't want code here bit-rotting.